### PR TITLE
More on links

### PR DESCRIPTION
**Pull request by BurdetteLamar**

### CODE REVIEW

File: doc/contributing/documentation_guide.md [hunk 1]
```diff
@@ -199,6 +199,36 @@ will be autolinked:
...
+#### +rdoc-ref+ Scheme
+
+Use the +rdoc-ref+ scheme for:
+
+- A link in core documentation to other core documentation.
+- A link in core documentation to documentation in a standard library package.
+- A link in a standard library package to other documentation in that same
+  standard library package.
+
+See section "+rdoc-ref+ Scheme" in {Links}[https://docs.ruby-lang.org/en/master/RDoc/MarkupReference.html#class-RDoc::MarkupReference-label-Links].
```
Comment (jeremyevans) [inline doc/contributing/documentation_guide.md:215]: Isn't this a link in the core documentation to documentation in a standard library package, and therefore, shouldn't it use `rdoc-ref` according to this?

**Review by jeremyevans — APPROVED** [review]
Looks good.  Please see inline question, and update the documentation if you agree.

File: doc/contributing/documentation_guide.md (hunk 1 above)
Comment (BurdetteLamar) [reply doc/contributing/documentation_guide.md:215]: I have not been able to get that to work.  Please make correction and merge.

Comment (jeremyevans) [reply doc/contributing/documentation_guide.md:215]: I'm not sure how to do so either.  Considering that, maybe we shouldn't be recommending use of rdoc-ref in a case where we don't know how to get it to work?

Comment (BurdetteLamar) [reply doc/contributing/documentation_guide.md:215]: @drbrain, can you help us out here?

**Pull request closed without merging**